integration-tests/services/basic/generated/TodoModule.d.ts

```diff
@@ -0,0 +1,7 @@
+export interface Todo {
```

Comment:
This file should have been ignored by .gitignore setting. Hmmm?

Reply:
Oh, I see. This particular test doesn't ignore the `generated` directory. Can you add that to one in `integration-tests`?